core/src/loader.rs

```diff
@@ -555,7 +532,8 @@ impl<'gc> Loader<'gc> {
                                 NEWEST_PLAYER_VERSION,
                                 uc,
                                 "broadcastMessage".into(),
-                                &["onLoadComplete".into(), Value::Object(broadcaster)],
+                                // TODO: Pass an actual httpStatus argument instead of 0.
+                                &["onLoadComplete".into(), clip.object(), 0.into()],
```

Comment:
Instead of `0`, should we use `200` as a temporary dummy status code?

Reply:
Seems like it should be 200 when loaded from a URL, and 0 otherwise (look for `httpStatus` in https://open-flash.github.io/mirrors/as2-language-reference/MovieClipLoader.html and also see expanded `mcl_loadclip` test in this branch). But I don't see an easy way to determine whether the fetch is offline/online.

Reply:
Ok, fine for now. Thank you!